webpack.config.js

```diff
@@ -0,0 +1,69 @@
+var webpack = require('webpack');
+var path = require('path');
+var fs = require('fs');
+
+var externalModules = [
+	'mkfifo',
+	'sqlite3',
+	'mysql2',
+	'pg',
+	'mariasql',
+	'mssql',
+	'mysql',
+	'strong-oracle',
+	'oracle',
+	'oracledb',
+	'pg-query-stream'
+]
+
+// We iterate over the node modules and mark all optional dependencies as external
```

Comment:
We only iterate over the top level node modules, not any nested or namespaced modules - it should probably handle those as well